spec/student_spec.rb

```diff
@@ -25,11 +28,12 @@
 
     it 'has an id that defaults to `nil` on initialization' do
       expect(josh.id).to eq(nil)
+      DB[:conn].execute("DROP TABLE students")
```

Comment:
Right right, okay.

I don't think that it makes sense to drop the table in this test -- it tightly couples this test and the test that comes after it, when we want to be able to run them in isolation.

We sometimes have to do weird things to "test" students' solutions on Learn, and this is one of those times.

Can we move the table-dropping to the `#create table creates the students table in the database` spec? So we'd have, e.g.,

```ruby
describe "#create_table" do
  it 'creates the students table in the database', :skip_before do
    DB[:conn].execute("DROP TABLE IF EXISTS students") # note the "IF EXISTS" -- see https://www.sqlite.org/lang_droptable.html
    Student.create_table
    table_check_sql = "SELECT tbl_name FROM sqlite_master WHERE type='table' AND tbl_name='students';"
    expect(DB[:conn].execute(table_check_sql)[0]).to eq(['students'])
  end
end
```

That way each of these tests can run independently, and if we need to add another spec or two to `#create_table`, we can simply move the DROP TABLE statement to a `before` (or `before(:each)`) block there.